packages/workbox-precaching/utils/createCacheKey.mjs

```diff
@@ -0,0 +1,64 @@
+/*
+  Copyright 2018 Google LLC
+
+  Use of this source code is governed by an MIT-style
+  license that can be found in the LICENSE file or at
+  https://opensource.org/licenses/MIT.
+*/
+
+import {WorkboxError} from 'workbox-core/_private/WorkboxError.mjs';
+
+import '../_version.mjs';
+
+// Name of the search parameter used to store revision info.
+const REVISION_SEARCH_PARAM = '_wbRevision';
+
+/**
+ * Converts a manifest entry into a versioned URL suitable for precaching.
+ *
+ * @param {Object} entry
+ * @return {string} A URL with versioning info.
+ *
+ * @private
+ * @memberof module:workbox-precaching
+ */
+export default function(entry) {
```

Comment:
Same comment as above, can we make this a named export?

Reply:
Changed.